src/modules/Dashboard/routes/routes.const.ts

```diff
@@ -13,6 +13,9 @@ const ParticipantConnections = lazy(
   () => import('modules/Dashboard/features/ParticipantConnections'),
 );
 const ParticipantSchedule = lazy(() => import('modules/Dashboard/features/ParticipantSchedule'));
+const ParticipantActivityDetails = lazy(
```

Comment:
Per my understanding, it's redundant variable, isn't it?

Reply:
Absolutely!